openpathsampling/deprecations.py

```diff
@@ -154,6 +154,16 @@ def version_tuple_to_string(version_tuple):
     deprecated_in=(1, 1, 0)
 )
 
+OPENMM_MDTRAJTOPOLOGY = Deprecation(
+    problem=("openpathsampling.engines.openmm.topology.MDTrajTopology "
+             "has been moved."),
+    remedy=("Import MDTrajTopology from openpathsampling.engines.topology "
+            "instead"),
```

Comment:
Could you add `MDTrajTopology` to the list of things imported into `openpathsampling.engines`? (i.e., edit `openpathsampling/engines/__init__.py`). Other topologies are in that namespace. That should be the canonical one. Then this diff should read:

```suggestion
    remedy=("Import MDTrajTopology from openpathsampling.engines instead."),
```